### ISSUE TITLE
[Feature]: Tell Sonarr unmonitor a series when it is all deleted

### ISSUE THREAD

**Issue by xiNeRoar** [shemanaev/jellyfin-plugin-media-cleaner#43]
### Describe the feature you'd like added

Sonarr cannot auto-unmonitor a series even if users delete all episodes in that series. using media cleaner with sonarr in this sense will causing numerous monitor series with 0 episodes on sonarr which is not desired.

Developer of Sonarr says it is a intended behavior because they won't know if users want to unmonitor the series or simply empty all the watched episode and ready for future season. So they won't implement auto unmonitor series function in forseeable future.

Since this is the case, I am wondering if media cleaner can add the ability to do so because users can always re-add series later. For example, a good current season ofa series doesn't guarantee a good future season, then if the users is maintaining a fairly huge library of series, he/she need to manually look through all the series and spot out the one that is unwanted and unmonitor them.

**Comment by JamsRepos** [comment]
Have you tried `/settings/mediamanagement` and using the 
<img width="755" alt="image" src="https://github.com/shemanaev/jellyfin-plugin-media-cleaner/assets/1347620/3b8734bd-bb58-4e54-9257-2b02900be8da">
setting? I do it, it works perfectly fine. I don't think the plugin needs to talk to sonarr.

**Comment by xiNeRoar** [comment]
> Have you tried `/settings/mediamanagement` and using the <img alt="image" width="755" src="https://private-user-images.githubusercontent.com/1347620/313024244-3b8734bd-bb58-4e54-9257-2b02900be8da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA0NzA5NzMsIm5iZiI6MTcxMDQ3MDY3MywicGF0aCI6Ii8xMzQ3NjIwLzMxMzAyNDI0NC0zYjg3MzRiZC1iYjU4LTRlNTQtOTI1Ny0yYjAyOTAwYmU4ZGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTVUMDI0NDMzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmYyMWRhZWE2NTEwMmI2ZDJiNjE1NmRhMjYxMGZkYzM1YzJkOTI3OWVkZGRkMzM4N2RlZjExYWVkYzYyYjk5YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.G9lIFHm-sNJqFcoWveqVu3zcqjNG_eke25xp1caUhSw"> setting? I do it, it works perfectly fine. I don't think the plugin needs to talk to sonarr.

Yes i do. But it will only delete and unmontior the episode. It will left the series itself monitored with zero episodes in it. This will cause a lot of unwanted series monitored if you regularly import a fair amount of series from list. And the only way to clear them is to go through all series in sonarr and click into them one by one to see which one has no episode since there is no visual cue in the list view to shows which series has no episode in it.

**Comment by JamsRepos** [comment]
I don't see much of a logistical issue if they're left in there unmonitored. They aren't going to do anything. There is a visual cue to see how many episodes there is within sonarr, look at the options top right and turn on progress bar. 
![image](https://github.com/shemanaev/jellyfin-plugin-media-cleaner/assets/1347620/1f64a9ea-49fb-44e4-aa2a-4056887cd96d)
